src/Avalonia.Themes.Fluent/Controls/ProgressBar.xaml

```diff
@@ -12,7 +15,6 @@
     <Setter Property="Background" Value="{DynamicResource SystemControlBackgroundBaseLowBrush}" />
     <Setter Property="BorderThickness" Value="{DynamicResource ProgressBarBorderThemeThickness}" />
     <Setter Property="BorderBrush" Value="{DynamicResource SystemControlHighlightTransparentBrush}" />
-    <Setter Property="Maximum" Value="100" />
```

Comment:
Maximum DirectProperty has default value of "100", so we don't need to set it in the styles.
@grokys is it expected that with DirectProperty value from setters is used instead of local value?
I remember that DirectProperty doesn't keep multiple values with different priorities, but preferring setter value instead of local is kind of unexpected here.

Reply:
Discussed, it's kind of unsupported behavior to use Setters with DirectProperty